ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java

```diff
@@ -233,10 +248,16 @@ public APIGatewayProxyResponseEvent handleRequest(
 
             if (errorObject.isPresent()) {
                 var accountInterventionStatus =
-                        ipvCallbackHelper.getAccountInterventionStatus(
-                                internalPairwiseSubjectId, auditContext);
-                if (configurationService.isAccountInterventionServiceActionEnabled()) {
-                    ipvCallbackHelper.doAccountIntervention(accountInterventionStatus);
+                        segmentedFunctionCall(
+                                "AIS: getAccountStatus",
+                                () ->
+                                        this.accountInterventionService.getAccountStatus(
+                                                internalPairwiseSubjectId, auditContext));
+                if (configurationService.isAccountInterventionServiceActionEnabled()
```

Comment:
Move this flag to within `logoutService.handleAccountInterventionLogout` to match the pattern above?